src/components/Text/Link.js

```diff
@@ -24,6 +24,14 @@ const LinkBase = styled(Base)`
   }
 `;
 
+const ReverseLinkBase = styled(LinkBase)`
+  color: ${themes.global.primary.reverse};
+
+  &:hover {
+    color: ${themes.global.primary.reverseLight};
```

Comment:
I'm not so sure about hard-coding these reversed colors... what are your thoughts on updating `getThemedFontColor` to support a `reverseColors` prop, so the API can stay consistent for other components? We can figure out a fallback for colors that lack a reverse hue.

Reply:
I see your point. But I don't think it's gonna be a dark theme for real. The design team already wants to use these colors in toast notifications no matter what theme is used. That's why I thought it makes sense to use props for switching between normal and reverse.

Reply:
Alright, cool